doc/man3/EVP_EncryptInit.pod

```diff
@@ -420,6 +423,10 @@ decrypted data buffer I<out> passed to EVP_DecryptUpdate() should have
 sufficient room for (I<inl> + cipher_block_size) bytes unless the cipher block
 size is 1 in which case I<inl> bytes is sufficient.
```

Comment:
(aside) I'm curious to know under what circumstances decryption can require a larger output buffer than the input size

Reply:
It's been a while since I last looked but I think it's related to the padding being decrypted and then trimmed.  This results in more memory writes.  I've no idea why it was done like this.